dss/api/subscriptions.py

```diff
@@ -164,8 +160,15 @@ def delete(uuid: str, replica: str):
     return jsonify({'timeDeleted': time_deleted}), requests.codes.okay
 
 
-def _register_percolate(es_client: Elasticsearch, uuid: str, es_query: dict, replica: str):
-    index_name = Config.get_es_index_name(ESIndexType.docs, Replica[replica])
+def _unregister_percolate(es_client: Elasticsearch, subscribed_indexes: list, uuid: str):
```

Comment:
Nit: list[str] would be more informative

Reply:
will fix